File: public/docs/_examples/dependency-injection/ts/app/heroes/hero-list.component.1.ts
// #docregion
import { Component }   from '@angular/core';

import { HEROES }      from './mock-heroes';

@Component({
  selector: 'hero-list',
  template: `
  <div *ngFor="let hero of heroes">
    {{hero.id}} - {{hero.name}}
  </div>
  `
})
export class HeroListComponent {
  heroes = HEROES;
}

File: public/docs/_examples/dependency-injection/dart/bin/heroes/hero-list.component.1.dart
// #docregion
import "package:angular2/core.dart";
import "hero.dart";
import "mock-heroes.dart";

@Component(
    selector: "hero-list",
    template: '''
  <div *ngFor="#hero of heroes">
    {{hero.id}} - {{hero.name}}
  </div>
  ''')
class HeroListComponent {
  var heroes = HEROES;
}
